### PR TITLE
Add support for Batches API endpoint.

### PR DESCRIPTION
**Pull request by bmcelhanon**

### CODE REVIEW

File: epay3.Web.Api.Sdk/Model/GetBatchesResponseMOdel.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+﻿using Newtonsoft.Json;
```
Comment (epay-mike-devlin) [inline epay3.Web.Api.Sdk/Model/GetBatchesResponseMOdel.cs]: Typo in the filename ("MOdel")

File: epay3.Web.Api.Sdk/Api/BatchesApi.cs [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        /// </summary>
+        /// <exception cref="epay3.Web.Api.Sdk.Client.ApiException">Thrown when fails to make API call</exception>
+        /// <param name="page"> (optional)</param> 
+        /// <param name="impersonationAccountKey"> (optional, default to )</param> 
+        /// <returns>ApiResponse of Object(void)</returns>
+        public GetBatchesResponseModel BatchesGet(int? page = null, string impersonationAccountKey = null)
+        {
+            var localVarPath = "/api/v1/Batches";
+    
+            var localVarPathParams = new Dictionary<String, String>();
```
Comment (epay-mike-devlin) [inline epay3.Web.Api.Sdk/Api/BatchesApi.cs:129]: Why do we use uppercase "String" throughout this project?

**Review by epay-mike-devlin — COMMENTED** [review]
Contingent on merging (internal) PRs 1978 and 1974.

File: epay3.Web.Api.Sdk/Model/GetBatchesResponseMOdel.cs (hunk 1 above)
Comment (bmcelhanon) [reply epay3.Web.Api.Sdk/Model/GetBatchesResponseMOdel.cs]: Uh... I definitely did this on purpose. Stands for Multiple Operation delete. Yep. Completely intentional...

**Pull request merged**